enterprise/backend/src/metabase_enterprise/api/routes.clj

```diff
@@ -28,6 +29,9 @@
   ;; and follow the convention.
   (compojure/context
    "/ee" []
+   (compojure/context
+    "/billing" []
+    billing/routes)
```

Comment:
Should I add a premium feature here? Unsure because it seems like anyone with a token should access this, but no other route is like this.

Reply:
This seems fine